styx-common/src/main/java/com/spotify/styx/model/WorkflowConfiguration.java

```diff
@@ -42,10 +42,10 @@
 
   Optional<String> offset();
 
-  Optional<String> dockerImage();
-
   Optional<String> commitSha();
 
+  Optional<String> dockerImage();
```

Comment:
What is the strategy for deprecating these?

Reply:
Step 1: Change model to allow both. This PR
Step 2: Remove old methods, keep tests that added in this PR to make sure that we still can read old json and read old entities in Datastore but converting old docker params to new DockerExecConf.

Reply:
👍